README.md

```diff
@@ -50,7 +50,6 @@ Rewrite `/contracts/data` to mask a site on github pages (note the redirect for
 
 ## tips
 - You can use sites like [regex101](https://regex101.com) to test your regular expressions.
-- You can use [tsv-pretty](https://ebay.github.io/tsv-utils-dlang/#tsv-pretty) to format the file to be more readable.
```

Comment:
This link yields a 404, but instead of removing it, would it be preferable to either link to the [command line utils](https://github.com/eBay/tsv-utils) or to a different hosted solution similar to the prior one?